__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const request = require('supertest');
+const app = require('../app');
+
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+
+const {articleData, commentData, topicData, userData } = require('../db/data/test-data/index')
+
+beforeEach(()=>{
+    return seed({topicData, userData, articleData, commentData })
+})
+afterAll(() => {db.end()});
+
+describe('app', ()=>{
```

Comment:
I think `api` is more than enough as the outer describe, you won't need to do any tests that fall outside of that so `app` as a wrapper can be removed